MdePkg/Include/Protocol/BootManagerPolicy.h

```diff
@@ -31,6 +31,13 @@
     0x113B2126, 0xFC8A, 0x11E3, { 0xBD, 0x6C, 0xB8, 0xE8, 0x56, 0x2C, 0xBA, 0xFA } \
   }
 
+// MU_CHANGE START: Introduce disk connect guid
+#define EFI_BOOT_MANAGER_POLICY_STORAGE_GUID \

```

Comment:
Is this planned to go to the UEFI spec?

Reply:
Yes, we are planning to take this to the UEFI spec and ideally push an open source implementation to EDK2's boot manager policy module

Reply:
Thanks. In that case, `EFI_` makes sense as a code first change and we'd get parity again with the upstream module.